adapters/madvertise/madvertise.go

```diff
@@ -100,14 +101,18 @@ func getImpressionExt(imp openrtb2.Imp) (*openrtb_ext.ExtImpMadvertise, error) {
 	var bidderExt adapters.ExtImpBidder
 	if err := json.Unmarshal(imp.Ext, &bidderExt); err != nil {
 		return nil, &errortypes.BadInput{
-			Message: err.Error(),
+			Message: fmt.Sprintf("ext not provided; ImpID=%s", imp.ID),
 		}
 	}
-
 	var madvertiseExt openrtb_ext.ExtImpMadvertise
 	if err := json.Unmarshal(bidderExt.Bidder, &madvertiseExt); err != nil {
 		return nil, &errortypes.BadInput{
-			Message: err.Error(),
+			Message: fmt.Sprintf("ext.bidder not provided; ImpID=%s", imp.ID),
```

Comment:
I'd recommend please append the original error message to this error message because an error unmarshaling `bidderExt.Bidder` doesn't always mean that it isn't provided. For example, an error here can also occur due to an invalid JSON.

Reply:
done [master 26c11086] append the original error message